pkg/sources/docker/metrics.go

```diff
@@ -16,11 +16,11 @@ var (
 	},
 		[]string{"source_name"})
 
-	dockerHistoryEntriesScanned = promauto.NewGaugeVec(prometheus.GaugeOpts{
+	dockerImageConfigsScanned = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
I don't really know if it is worth keeping this metric as in most cases it will be the same as `dockerImagesScanned`